lib/engine/ability/sell_to_bank.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
+  module Ability
+    class SellToBank < Base
```

Comment:
only certain companies can be sold to the bank?

Reply:
No, all private companies can. I need an ability so it shows up during the Stock round. Is there another way of doing that?

Reply:
yes, i don't think you need entities, just have them show up in the stock round through a method in the step, sellable_companies or whatever and remove all these ability references

Reply:
But it needs to show up with the other abilities in the UI, doesn't it? How else would it appear?